src/features/components/molecules/layout-overlay/cfd-warning-banner/index.tsx

```diff
@@ -1,29 +1,108 @@
-import React from 'react'
-import { cfd_banner_container } from './cfd-warning-banner.module.scss'
+import React, { useState } from 'react'
+import * as styles from './cfd-warning-banner.module.scss'
 import { Localize } from 'components/localization'
+import Arrow from 'images/svg/arrow_expandable.svg'
 import Container from 'features/components/atoms/container'
 import { loss_percent } from 'common/constants'
+import Flex from 'features/components/atoms/flex-box'
+import Link from 'features/components/atoms/link'
+import Image from 'features/components/atoms/image'
 import Typography from 'features/components/atoms/typography'
 import useRegion from 'components/hooks/use-region'
 import usePpc from 'features/hooks/use-ppc'
 
 const CfdWarningBanner = () => {
     const { is_ppc } = usePpc()
     const { is_eu, is_cpa_plan } = useRegion()
+    const [expanded, setExpanded] = useState(false)
 
     if (is_ppc || is_eu || is_cpa_plan) {
+        const toggleExpansion = () => {
+            setExpanded(!expanded)
```

Comment:
```suggestion
            setExpanded(prev => !prev)
```

Since, useState works asynchronously and here current state depends on previous one. So, we should toggle this state this way.